pyproject.toml

```diff
@@ -13,7 +13,7 @@ classifiers = [
 description = "Create an IOC from a PandA"
 dependencies = [
     "setuptools>=64",
-    "numpy",
+    "numpy<2",
```

Comment:
Probably worth adding the p4p issue number here too, to explain why it's pinned.